docs/install.md

```diff
@@ -1,9 +1,18 @@
 # Installing the Support-Lib
 
-## conan
+## Conan
 
-!!! bug "TODO"
+The library is available at [conan-center](https://conan.io/center/djinni-support-lib):
 
-## hunter
+```sh
+conan install djinni-support-lib/0.0.1@
+```
 
-!!! bug "TODO"
+### Options
+
+| Option | Values | Description |
+| ------ | ------ | ----------- |
+| `shared` | `True`, `False` | Wether to build as shared library. Default: `False` |
+| `fPIC` | `True`, `False` | If position-independent code should be generated. Default: `True` |
+| `target` | `jni`, `objc`, `auto` | The library has different targets for usage with Java or Objective-C. By default (`auto`) the target is determined automatically depending on the target OS (`iOS` → `objc`, `Android` → `jni`). Set this explicitly if you want to build for `macOS`/`Windows`/`Linux`, because on these platforms both targets may be a valid option! |
+| `system_java` | `True`, `False` | The library needs to link against the JNI (Java Native Interface) if `target` is `jni`. By default (`True`), `zulu-openjdk/11.0.8` will be installed from conan center for this. Set to `False` to use the systems Java installation instead.  |
```

Comment:
I have tried to document the two custom conan options a bit.

Reply:
the `fPIC` topic is quite complex, because they are kind of auto managed .. and there is Windows .... where it usually gets deleted .... :-) 
but I think it's OK to have it like that

Reply:
I've literally googled what it does, all I know it that it is a "default option" that ever lib has... it may be better to remove the documentation here instead of adding inaccurate information?

Reply:
leave it but make it more generic, like 

`True, Follows the default conan behaviour` 

;-)

Reply:
you can see the actual behaviour here:
https://github.com/conan-io/conan-center-index/blob/41d4fbb8c218f9d938ccb8b9d888bff52002e62e/recipes/djinni-support-lib/all/conanfile.py#L55
(the following ~6 lines)

Reply:
I've adjusted the fPIC documentation with your recommended description.